nemo/core/classes/modelPT.py

```diff
@@ -190,14 +190,29 @@ def restore_from(cls, restore_path: str):
                 model_weights = path.join(tmpdir, _MODEL_WEIGHTS)
                 conf = OmegaConf.load(config_yaml)
                 OmegaConf.set_struct(conf, True)
+                cls.update_config_with_specific_artifacts(config=conf, artifacts_dir=tmpdir)
```

Comment:
This needs conf = cls.update... to match return signature of the method.

Reply:
fixed